plugins/org.eclipse.n4js.model/model/N4JS.xcore

```diff
@@ -2287,6 +2287,9 @@ class JSXElement extends Expression, JSXChild {
 	contains JSXElementName jsxClosingName
 }
 
+class JSXFragment extends Expression, JSXChild {
```

Comment:
Can the class `JSXElement` **extend** this new class `JSXFragment`? In case it can: It would be nice to introduce a new private method in `CFEChildren` to reduce copied code.

Reply:
I am afraid not since conceptually JSXElement is not a special case of JSXFragment...

Reply:
Done. Introduce JSXAbstractElement as basic class for JSXFragment and JSXElement